examples/vpc-scenario-2/main.tf

```diff
@@ -14,6 +19,11 @@ variable "region" {
   default     = "us-east-2"
```

Comment:
Might as well make `region` a variable while adding `vpc_cidr`.

Reply:
So do you mean that instead of having a separate region variable, like it currently it, that `region` should be nested and set within the `vpc_cidr` variable?

Reply:
No, that comment of mine was confused. I think I mistakenly thought I saw the AWS provider using a hardcoded region, but I don't see that and am not sure how I got off track. Sorry for the noise!